__tests__/app.test.js

```diff
@@ -52,20 +52,62 @@ describe('# GET REQUESTS', () => {
                 .get('/api/articles/1')
                 .expect(200)
                 .then(res => {
-                    const articles = res.body.articles;
-                    articles.forEach(article => {
-                        
-                        // check object properties
-                        expect(article).toMatchObject({
+                    const article = res.body.article;
+
+                    expect(article).toMatchObject({
                         author: expect.any(String),
                         title: expect.any(String),
                         article_id: 1,
                         body: expect.any(String),
                         topic: expect.any(String),
                         created_at: expect.any(String),
-                        votes: expect.any(Number)});
-                    });
+                        votes: expect.any(Number),
+                        comment_count: 11});
+                });
+            });
+
+        test('404: ID not found when given valid non-existent id', () => {
+            return request(app)
+                .get('/api/articles/100000')
+                .expect(404)
+                .then(res => {
+                    console.log(res.body)
```

Comment:
Always a good idea to remove your console.logs as soon as you don't need them anymore